docs/customisation/configuration.md

```diff
@@ -1,12 +1,14 @@
 # Configuration
 
-The configurable properties for Inji can be found at [inji-default.properties](https://github.com/mosip/mosip-config/blob/qa-1201-B2/inji-default.properties). This property file is maintained as one for each deployment environment. On [this](https://github.com/mosip/mosip-config) repository, each environment configuration is placed in a corresponding branch specific to that environment.
+## Configuration for Inji
+
+The configurable properties for Inji can be found at [inji-default.properties](https://github.com/mosip/mosip-config/blob/qa-inji/inji-default.properties). This property file is maintained as one for each deployment environment. On [this](https://github.com/mosip/mosip-config) repository, each environment configuration is placed in a corresponding branch specific to that environment.
 
 > Refer to [inji-default.properties](https://github.com/mosip/mosip-config/blob/collab/inji-default.properties) of Collab environment.
 
 These values will be used by Inji via Mimoto. Mimoto loads all the configurations and exposes an API which is used by the Inji application to fetch and store the configurations in the local storage.
 
-API used to fetch these configurations: `https://api.mosip.io/residentmobileapp/allProperties`
+API used to fetch these configurations: `https://api.mosip.io/v1/mimoto/residentmobileapp/allProperties`
```

Comment:
Is the URL change not dependent on server & setup config? Should we have such examples with the hostname if these ENVs aren't up always?

Reply:
This is the production env which we expect to be up.
and yes this url is dependent on deployment env.